CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Documentation on writing examples and running doctests
 ### Changed
 - Restructured contributing documentation
+- Added npm_audit.py and test_npm_audit.py
```

Comment:
- If we're adding something, we want to put it under the `Added` section (currently it's under `Changed`)
  - We won't need to start the sentence with `Added`, because we'll be under the `Added` section.
- Let's describe what the significance of the change is, this just tells us that we've added two files.
For example, last time the addition to the changelog told us what tool was added for what language.

```markdown
- shouldi got an operation to run golangci-lint on Golang code
```

This time, we might want to say something about what `npm audit` does, we're running `npm audit` so that we can detect vulnerabilities in dependencies of JavaScript applications.